src/v2/view-builder/utils/AuthenticatorUtil.js

```diff
@@ -184,6 +184,7 @@ const getAuthenticatorData = function(authenticator, isVerifyAuthenticator) {
       description: isVerifyAuthenticator
         ? authenticator?.relatesTo?.displayName
         : loc('oie.custom.app.authenticator.description', 'login', [authenticator.label]),
+      noTranslateClassName: isVerifyAuthenticator ? 'no-translate' : '',
```

Comment:
Seems unrelated to PR - is this on purpose?

Reply:
Yes, this is a separate issue I found. Must be hidden by false positives on english leaks. I will cherry-pick this over to a separate PR

Reply:
https://github.com/okta/okta-signin-widget/pull/2564